bigbluebutton-html5/imports/ui/components/whiteboard/utils.js

```diff
@@ -212,11 +212,90 @@ const mapLanguage = (language) => {
   }
 };
 
+const getFontStyle = (style) => {
```

Comment:
I'd recommend adding a comment saying where in tldraw these functions were copied from - that'll help folks who come by later and might have to update things to keep it in sync.